### DISCUSSION TITLE
How to output and check mesh partition?

### DISCUSSION THREAD

**Discussion by Minjiang-Zhu** [idaholab/moose#26237] [Q&A General] [answered]
I am using mesh adaptivity, yet the simulation is slower than expected. I want to check how is the mesh partitioned for each processor.

**Comment by Minjiang-Zhu** [comment, marked as answer]
I found ProcessorIDAux.

**Reply by GiudGiud** [reply]
there's a debug option to show the partitions too. It relies on processorIDaux though, so it s the same

**Reply by Minjiang-Zhu** [reply]
Good to know. Thank you!
